### PR TITLE
finished writing tests for get_r2

### PR DESCRIPTION
**Pull request by jentsang**
Finished writing the tests for get_r2() function

### CODE REVIEW

File: tests/unit/test_get_r2.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""
+A test module that tests the get_r2() function.
+Note: these tests are written with the assitance of LLMs.
+"""
+
+from reportrabbit.get_r2 import get_r2
```
Comment (ruthyankson) [inline tests/unit/test_get_r2.py]: The module file is currently named `r2`, but the test imports `reportrabbit.get_r2`. This caused import errors when I tested the code. Could you please align the import path with the file name?

**Review by raghav9048 — COMMENTED** [review]
Hi Jennifer, great work overall. 
The existing test cases cover the main expected behaviors for R square.

As a possible improvement, we can maybe also consider adding tests for edge cases such as shape/length mismatches, inputs containing NaN or infinite values and scenarios where y_true has zero variance. 
Overall, this looks solid. Thanks!
